lib/utils/ajvErrorMessageBuilder.test.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const expect = require('chai').expect;
+const ajvErrorMessageBuilder = require('./ajvErrorMessageBuilder');
+
+describe('#ajvErrorMessageBuilder', () => {
+  it('should build user friendly message for unsupported function event', () => {
+    const errors = [
+      {
+        keyword: 'additionalProperties',
```

Comment:
It'll be nice to use `runServerless`, and inspect that output contains expected error string.

We can also provoke invalid function event error that way, and inspect that expected nicer error message landed in output.

Only such tests will give us a guarantee that those error mapping are in effect and indeed surface in output dedicated for user

Reply:
I would propose error message testing to be placed in `lib/configSchema/index.test.js` file. This file has tests that use `runServerless` and I am planning to add expected error messages to the test cases to make sure they are shown to user.

As for this file, I see it to be about converting array of AJV's error objects to a single error message string. Manipulations performed here are trivial and do not require other parts of the framework. 

So I'd prefer to keep tests in this file without `runServerless` util.

If you have another thing in mind about this, let me know

Reply:
Yes, it could be there (or by `lib/classes/ConfigSchemaHandler`)